bears/general/InvalidLinkBear.py

```diff
@@ -85,12 +89,18 @@ def find_links_in_file(file, timeout, link_ignore_regex, link_ignore_list):
                         fnmatch(link, link_ignore_list)):
                     if link.startswith(('hg+', 'bzr+', 'git+', 'svn+')):
                         link = InvalidLinkBear.parse_pip_vcs_url(link)
-                    code = InvalidLinkBear.get_status_code(link, timeout)
+                    host = urlparse(link).netloc
+                    code = InvalidLinkBear.get_status_code(link,
+                                                           network_timeout[
+                                                               host])
                     yield line_number + 1, link, code
 
+    return_default_timeout = lambda: DEFAULT_TIMEOUT  # pragma: no cover
```

Comment:
Explanation for using `no cover` given below :point_down: 

@Makman2 you are on!
Any suggestions for getting the lambda covered, while declaring it in the run function itself, this was the workaround I found :wink: